model/src/guild/mod.rs

```diff
@@ -25,17 +25,35 @@ mod unavailable_guild;
 mod vanity_url;
 mod verification_level;
 mod widget;
+mod widget_settings;
 
 pub use self::{
-    ban::Ban, default_message_notification_level::DefaultMessageNotificationLevel, emoji::Emoji,
-    explicit_content_filter::ExplicitContentFilter, info::GuildInfo, integration::GuildIntegration,
-    integration_account::IntegrationAccount, integration_application::IntegrationApplication,
-    integration_expire_behavior::IntegrationExpireBehavior, member::Member, mfa_level::MfaLevel,
-    nsfw_level::NSFWLevel, partial_guild::PartialGuild, partial_member::PartialMember,
-    permissions::Permissions, premium_tier::PremiumTier, preview::GuildPreview, prune::GuildPrune,
-    role::Role, role_tags::RoleTags, system_channel_flags::SystemChannelFlags,
-    unavailable_guild::UnavailableGuild, vanity_url::VanityUrl,
-    verification_level::VerificationLevel, widget::GuildWidget,
+    ban::Ban,
+    default_message_notification_level::DefaultMessageNotificationLevel,
+    emoji::Emoji,
+    explicit_content_filter::ExplicitContentFilter,
+    info::GuildInfo,
+    integration::GuildIntegration,
+    integration_account::IntegrationAccount,
+    integration_application::IntegrationApplication,
+    integration_expire_behavior::IntegrationExpireBehavior,
+    member::Member,
+    mfa_level::MfaLevel,
+    nsfw_level::NSFWLevel,
+    partial_guild::PartialGuild,
+    partial_member::PartialMember,
+    permissions::Permissions,
+    premium_tier::PremiumTier,
+    preview::GuildPreview,
+    prune::GuildPrune,
+    role::Role,
+    role_tags::RoleTags,
+    system_channel_flags::SystemChannelFlags,
+    unavailable_guild::UnavailableGuild,
+    vanity_url::VanityUrl,
+    verification_level::VerificationLevel,
+    widget::{GuildWidget, GuildWidgetChannel},
```

Comment:
let's only re-export widget since this is not actually exporting all the types from the module
```suggestion
    widget::GuildWidget,
```